h/accounts/views.py

```diff
@@ -649,9 +649,9 @@ def reset_password_email(user, reset_code, reset_link):
     }
 
 
-def reset_password_link(request, reset_code):
+def account_reset_link(request, reset_code):
     """Transform an activation code into a password reset link."""
```

Comment:
We might want to change the wording in the docstring as well to say "account reset link".